### PR TITLE
Development

### PR DESCRIPTION
**Pull request by StarFluxGames**

### CODE REVIEW

**Comment by Ikealtea** [comment]
What was the original intent with the "fun" code..? Why have stuff like that in such a commonly used mod?

**Comment by nitehawk** [comment]
Can this also be removed?

https://github.com/KitchenMods/KitchenLib/blob/5a035830bf0b2e18eb459f6beb607fcc16a4178d/KitchenLib/src/FeatureFlags.cs#L17-L23

**Comment by Ikealtea** [comment]
> Can this also be removed?
> 
> https://github.com/KitchenMods/KitchenLib/blob/5a035830bf0b2e18eb459f6beb607fcc16a4178d/KitchenLib/src/FeatureFlags.cs#L17-L23

What could possibly be wrong with some auto invite code? LGTM! /s

File: KitchenLib/Changelogs/v0.7.6.MD [hunk 1]
```diff
@@ -0,0 +1,3 @@
+﻿# Release Notes
+
+Changelogs not current at this time, will update later.
```
Comment (Ikealtea) [inline KitchenLib/Changelogs/v0.7.6.MD:3]: Documentation is important. I think we need to add information to the change log about what exactly this PR is doing.

Comment (StarFluxGames) [reply KitchenLib/Changelogs/v0.7.6.MD:3]: Didn’t have time to write documentation as it was 6am, although with the nature of the platform you can see exactly what was changed in the commit.

**Comment by StarFluxGames** [comment]
> 

If you proceed to read through the source code of KitchenLib, this snippet of code hasn't been connected to anything for quite a while and is not functional. It will still be removed.

**Pull request merged**